<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #f2efff;</string>
  </property>
  <widget class="QFrame" name="login_frameBase">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>831</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#login_frameBase{
	background-color: #FFFFFF;
	border-radius: 20px;
	border: 2px solid #ffffff;
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QFrame" name="login_frameBlue">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>0</y>
      <width>281</width>
      <height>501</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#login_frameBlue{
	background-color: #141c52;
	border: 2px solid #141c52;
	border-top-left-radius: 20px;  
    border-bottom-left-radius: 20px;
    border-top-right-radius: 0px; 
    border-bottom-right-radius: 0px;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="login_labelTitle2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #141c52;</string>
    </property>
    <property name="text">
     <string>PROFILING DATABASE SYSTEM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_buttonLogin">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#login_buttonLogin {
    background-color: #141c52;
    color: white;
    border-radius: 20px;
    padding: 10px; /* Optional: Adjusts spacing inside the button */
}
#login_buttonLogin:hover {
    background-color: #1a237e; /* Slightly lighter blue when hovered */
}
#login_buttonLogin:pressed {
    background-color: #0f1442; /* Darker blue when clicked */
}</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_fieldEmp_id">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>259</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#login_fieldEmp_id{
	border: 1px solid #b3b3b3;  /* Black border */
    border-radius: 5px;        /* Rounded corners */
    padding: 5px;               /* Inner spacing */
    background-color: #f2efff;  /* Gray background */
    color: #000000; 
}

</string>
    </property>
    <property name="placeholderText">
     <string>Please Enter ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_labelPin">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#login_labelPin{
	background-color:transparent;
	color:#4f4f4f;
}</string>
    </property>
    <property name="text">
     <string>PIN:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_fieldPin">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#login_fieldPin{
    border: 1px solid #b3b3b3;  /* Black border */
    border-radius: 5px;        /* Rounded corners */
    padding: 5px;               /* Inner spacing */
    background-color: #f2efff;  /* Gray background */
    color: #000000; 
}
</string>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
    <property name="placeholderText">
     <string>XXXXXX</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_labelTitle1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
color: #141c52;</string>
    </property>
    <property name="text">
     <string>BARANGAY MARIGONDON</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_labelEmp_id">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>259</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#login_labelEmp_id{
	background-color:transparent;
	color:#4f4f4f;
}</string>
    </property>
    <property name="text">
     <string>Employee ID:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="login_imageLogo">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#login_imageLogo{	
	background-color: transparent;
	color: #ffffff;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Assets/BARANGAY MARIGONDON SYSTEM.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="login_labelRights">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>470</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#login_labelRights{
	background-color: transparent;
	border-color: transparent;
	color: #7a7a7a;
}

</string>
    </property>
    <property name="text">
     <string>Â© 2025 Barangay Marigondon. All Rights Reserved.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <zorder>login_labelTitle2</zorder>
   <zorder>login_buttonLogin</zorder>
   <zorder>login_fieldEmp_id</zorder>
   <zorder>login_labelPin</zorder>
   <zorder>login_fieldPin</zorder>
   <zorder>login_labelTitle1</zorder>
   <zorder>login_labelEmp_id</zorder>
   <zorder>login_imageLogo</zorder>
   <zorder>login_labelRights</zorder>
   <zorder>login_frameBlue</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
